.github/workflows/windows-ci.yml

```diff
@@ -1,40 +0,0 @@
-name: Windows
-
-on:
-  push:
-    branches: [ main ]
-    paths-ignore:
-    - '**.md'
-  pull_request:
-    branches: [ main ]
-    paths-ignore:
-    - '**.md'
-
-jobs:
-  build-test:
-    runs-on: windows-latest
```

Comment:
@odubajDT I'm OK with consolidating jobs, but we need to run a bujild on `windows-latest` to ensure the integrity of the `net462` build.

Reply:
good point, will fix

Reply:
Actually running e2e tests on windows might be a little bit problematic due to that container operations are only supported on Linux runners. Therefore we are unable to start the flagd container. I guess it is ok when the e2e tests are ran only on linux.

Reply:
Fixed, so in the current state the following tests are ran:

linux: unit tests with .Net 6 and .Net 7 + E2E tests
windows: unit tests with .Net 6, .Net 7 and .Net 4.6.2

Running e2e tests on windows might be a little bit problematic due to that container operations are only supported on Linux runners. Therefore we are unable to start the flagd container. I guess it is ok when the e2e tests are ran only on linux.

Hope this meets the requirements.

Reply:
I was worried because I didn't see test commands specifically targeting the `bin\Release\net462\` folder, but it seems like they are being picked up automatically in the windows build:

```
Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
D:\a\dotnet-sdk\dotnet-sdk\test\OpenFeature.Tests\bin\Release\net6.0\OpenFeature.Tests.dll
[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.3+1b45f5407b (64-bit .NET 6.0.16)
[xUnit.net 00:00:07.30]   Discovering: OpenFeature.Tests
[xUnit.net 00:00:07.54]   Discovered:  OpenFeature.Tests
[xUnit.net 00:00:07.54]   Starting:    OpenFeature.Tests
```

So I think we're good!